src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -35,10 +35,16 @@
     }
   }
 
-  &:focus {
+  &:focus-visible {
```

Comment:
Is there preferred practices for wrapping in `@supports` vs not and just having fallback only? i.e. should we be wrapping this with `@supports` since the effect would be the same

Reply:
Discussed async, don't think this need wrapping since browser will ignore if unsupported, resolving conversation